contracts/contracts/oracle/OracleRouter.sol

```diff
@@ -87,17 +85,57 @@ contract OracleRouter is OracleRouterBase {
         ) {
             // Chainlink: CVX/USD
             return address(0xd962fC30A72A84cE50161031391756Bf2876Af5D);
+        } else if (
+            asset == address(0xae78736Cd615f374D3085123A210448E74Fc6393)
```

Comment:
Any reason for explicitly casting to address? Isn't it already an address?

Reply:
You are correct. Removed.